boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm33_defconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_PINCTRL=y
+CONFIG_FLEXSPI_CONFIG_BLOCK_OFFSET=0x400
```

Comment:
Do we need to set this again? It is set at SOC level

Reply:
No need, I will delete it. Sorry, this content was copied from RT1170.